src/zimit/zimit.py

```diff
@@ -558,6 +558,13 @@ def get_node_cmd_line(args):
         "logging",
     ]:
         value = getattr(args, arg)
+        if arg == "userAgent":
+            # strip leading whitespace which are not allowed on some websites
```

Comment:
`str.strip()` strips whitespaces on both side of the string.

Reply:
Yep, this is what is finally intended 
This PR first comment explains why, but I have to admit I changed this between the opening of PR and the submission to review because I realized it was needed

Reply:
Oh but comment is still wrong, I got it